### PR TITLE
feat: Mobile Layout, Dispaly SubMenu

### PR DESCRIPTION
**Pull request by AbdullahAli420**
## Changes proposed
See video!

## Screenshots
Old:-

https://github.com/thecyberworld/TheCyberHUB/assets/118581404/0353ea16-acc8-4bbf-b69e-cdb0c8acc18c

Fixed:-

https://github.com/thecyberworld/TheCyberHUB/assets/118581404/ec557eb2-c3e8-4bdb-acaf-542964903133

## Note to reviewers
Feel free to tell me any changes you want!

## Code of Conduct

- [ ] By submitting this pull request, I confirm I've read and complied with the [CoC](https://github.com/thecyberworld/Support/blob/main/CODE_OF_CONDUCT.md) 🖖

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

---

You can also join our [Discord](https://discord.gg/QHBPq6xP5p) community.
Feel free to check out other cool repositories of the [Thecyberworld](https://github.com/thecyberworld).
Join the Thecyberworld GitHub Organisation by raising an [issue](https://github.com/thecyberworld/Support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) (you will be sent an invitation).

### CODE REVIEW

**Comment by ArkadiK94** [comment]
@AbdullahAli420 Very important, before submitting your code. You should check the console (you have two errors):

image

**Comment by ArkadiK94** [comment]
before
image

after
image

the menu on media range : 822-1150 not responsive as it was before your changes

in addition, your new sub tabs not happens on media range : 769-821
769:
image
768:
image

**Review by ArkadiK94 — CHANGES_REQUESTED** [review]
@AbdullahAli420 Thank you for your contribution, please, fix the comments that I wrote.

**Comment by ArkadiK94** [comment]
@AbdullahAli420 Great, I tried it out, and one more issue,
when I click first tab the sub tabs open, when I click different tab the sub tabs of previous tab close but the sub tabs of the different tab are not opened

https://github.com/thecyberworld/TheCyberHUB/assets/76536506/ff0ccf0f-4693-4572-b38b-b37be14b7b34

**Comment by AbdullahAli420** [comment]
Done check that.

File: src/components/Header/Dropdowns/DropdownElements.jsx [hunk 1]
```diff
@@ -61,13 +69,23 @@ export const DropdownItemContainer = styled.div`
...
+    margin: 0;
 
     &:hover {
         cursor: pointer;
     }
 
-    @media screen and (max-width: 768px) {
-        max-width: 40%;
+    @media screen and (max-width: 821px) {
+        // width: min-content;
```
Comment (ArkadiK94) [inline src/components/Header/Dropdowns/DropdownElements.jsx:82]: if you don't use it, remove it.

File: src/components/Header/Navbar/NavbarElements.jsx [hunk 2]
```diff
@@ -40,6 +41,7 @@ export const NavbarContainer = styled.div`
     z-index: 1;
     width: 100%;
     max-width: 1500px;
+    // display: none;
```
Comment (ArkadiK94) [inline src/components/Header/Navbar/NavbarElements.jsx:44]: if you don't use it, remove it , I am writing here, but I refer to all commented styles that you don't use.

**Review by ArkadiK94 — APPROVED** [review]
@AbdullahAli420 Looks great and thank you for your contribution 👍 
I have one last minor comment.

**Comment by ArkadiK94** [comment]
@AbdullahAli420 I will merge the pr because it is not a big deal, but for next time, please don't leave commented styles that you don't need to use :)

**Comment by AbdullahAli420** [comment]
Ok I will take care of that.

**Pull request merged**